app/models/user.rb

```diff
@@ -604,6 +632,23 @@ def saw_notification_id(notification_id)
     end
   end
 
+  def bump_last_seen_reviewable!
+    max_reviewable_id = Reviewable
+      .unseen_list_for(self, preload: false, limit: 1)
+      .except(:order)
+      .order(id: :desc)
+      .pluck_first(:id)
```

Comment:
```suggestion
    max_reviewable_id = Reviewable
      .unseen_list_for(self, preload: false)
      .maximum(:id)
```

I think this works the same without the need for ordering or limiting.